src/components/pages/Home/HomeContainer.js

```diff
@@ -15,29 +21,33 @@ function HomeContainer({ LoadingComponent }) {
     memoAuthService
       .getUser()
       .then(info => {
-        // if user is authenticated we can use the authService to snag some user info.
-        // isSubscribed is a boolean toggle that we're using to clean up our useEffect.
         if (isSubscribed) {
-          setUserInfo(info);
+          postUserId(info.sub);
+          postProfile(info);
         }
       })
       .catch(err => {
         isSubscribed = false;
-        return setUserInfo(null);
+        console.log(err);
```

Comment:
We should remove console logs from the components

Reply:
Done!